### PR TITLE
Implemented  Dark mode and app responsive

### PR DESCRIPTION
**Pull request by dev-prabhat**
Implemented dark mode and make app responsive

### CODE REVIEW

**Review by humzahasan — APPROVED** [review]
- Code changes looks good!
- App is responsive and looks good in both dark and light theme


Good to merge!

File: src/components/SingleNote/SingleNote.js [hunk 1]
```diff
@@ -33,20 +34,23 @@ export const SingleNote = ({note, setEditModal, isHomePage = false, isTrashPage
...
    
     return(
-        <div className="single-note margin-xs padding-xs" style={{backgroundColor:color}}>
-            <h2 className="head-md">{title}</h2>
+        <div 
+            className={`single-note margin-xs padding-xs  ${theme === "light" ? "light-theme" : "dark-theme"} `} 
+            style={{backgroundColor:color}}
+        >
+            <h2 className="note-title">{title}</h2>
             <div className="text-sm padding-sm" dangerouslySetInnerHTML={{ __html: DOMPurify.sanitize(body) }}></div>
```
Comment (iRohitGaur) [inline src/components/SingleNote/SingleNote.js:42]: suggestion: give this a read: https://javascript.plainenglish.io/sanitize-your-html-and-keep-its-style-in-react-1ff9fdd8e3cc

**Review by iRohitGaur — APPROVED** [review]
Nice work! Good to merge 👍🏻

File: src/components/SingleNote/SingleNote.js (hunk 1 above)
Comment (dev-prabhat) [reply src/components/SingleNote/SingleNote.js:42]: Thanks for this suggestion , i have already implemented dompurify

**Pull request merged**